### ISSUE TITLE
Table field configuration for maximum number of columns

### ISSUE THREAD

**Issue by oksana-c** [department-of-veterans-affairs/va.gov-cms#2933]
## Description

As a table author/editor, I want to keep my table usable and readable by not adding more than 12 columns of data per table.

<img width="1041" alt="Screen Shot 2020-09-16 at 2 09 09 PM" src="https://user-images.githubusercontent.com/16477724/93387337-4a774e80-f826-11ea-9eeb-8d10e4ace091.png">

## Acceptance Criteria

- [ ] content authors can not add more than 12 columns per table
- [ ] constraining columns to a max number should not affect any existing tables

## Implementation

* patch for the `tablefield` contrib module
     * Tablefield module has a config form element that allows to set a maximum amount of columns per table

**Comment by swirtSJW** [comment]
I am not sure how we came up with 12 . vs 10 or 20.   Either way it is likely bad.

**Comment by EWashb** [comment]
I agree @swirtSJW. Not sure where they were going with this one. I don't even think the current reality deems this within scope and would be ok to close it. We would need to do some additional research/spelunking for the current state before we decide 12 is the right choice.
